components/track/TrackContributors.vue

```diff
@@ -0,0 +1,23 @@
+<script setup lang="ts">
+import type { TrackModel } from "@bcc-code/bmm-sdk-fetch";
+
+const showMediaPlayer = inject<boolean>("showMediaPlayer");
+
+defineProps<{
+  track: TrackModel;
+}>();
+</script>
+
+<template>
+  <div class="bg-background-2 rounded-lg">
+    <div v-for="(contributor, index) in track?.contributors" :key="index">
+      <NuxtLink
+        class="rounded-lg text-label-1 flex flow-row gap-3 p-2 px-5 hover:bg-background-3"
```

Comment:
hover:bg-background-3 looks weird since it gets white. I would expect a darker color. I don't think Søren has provided a color yet, though.

Reply:
Søren just added that it should be hover:bg-label-separator

Reply:
done

Reply:
done